flashinfer/trace_apply/runtime.py

```
@@ -0,0 +1,442 @@
+        produced_active = []
+        for i, jk in enumerate(out_keys):
+            val = produced[i] if i < len(produced) else None
+            if val is not None:
+                buf = dest_buf(jk)
+                if buf is not None and isinstance(val, torch.Tensor) and val.data_ptr() != buf.data_ptr():
+                    buf.copy_(val)
+                    val = buf
+            if jk in active_keys:
+                produced_active.append(val)
+        if any(v is None for v in produced_active):
+            missing = [jk for jk, v in zip(active_keys, produced_active) if v is None]
```

Ruff (0.15.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)